src/Umbraco.Core/PropertyEditors/PropertyEditor.cs

```diff
@@ -13,6 +14,7 @@ namespace Umbraco.Core.PropertyEditors
     /// <remarks>
     /// The Json serialization attributes are required for manifest property editors to work
     /// </remarks>
+    [DebuggerDisplay("{DebuggerDisplay(),nq}")]

```

Comment:
You can just put in the expression: 

     string.Format("Name: {0}, Alias: {1}, IsParameterEditor: {2}", Name, Alias, IsParameterEditor)

If people want to 'override' this, then they can apply their own attribute. 

What is the 'nq' part though?